lib/yi_appium_caps_util.rb

```diff
@@ -167,6 +172,7 @@ def update_ios_caps (output_data)
           puts 'IP Address: ' + ip
           if (ip == "")
             ipcounter +=1
+            sleepSec +=5
           else
             #Found it!  
             ipcounter = 999
```

Comment:
I know its not changed in your PR, but do you think a "return" would work just as well as setting the counter to 999? That way we wouldn't have to check the loop condition again.